### PR TITLE
Update FlushAt to 10

### PR DESCRIPTION
**Pull request by mathur07**

### CODE REVIEW

**Comment by mathur07** [comment]
we can update to any number, 3 is just an arbitrary number

**Comment by mathur07** [comment]
for ref: https://segment.com/docs/connections/sources/catalog/libraries/server/node/#configuration

File: src/utils/segmentInitializer.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export namespace SegmentInitializer {
         ref: https://segment.com/docs/connections/sources/catalog/libraries/server/node/#configuration
         */
       let analytics: Analytics = new Analytics(segmentWriteKey, {
-        flushAt: 1,
+        flushAt: 3,
```
Comment (fbricon) [inline src/utils/segmentInitializer.ts:29]: flushAt: 10

Comment (mathur07) [reply src/utils/segmentInitializer.ts]: done

**Comment by fbricon** [comment]
thx @mathur07

**Pull request merged**